### ISSUE TITLE
Default seek bar is not working correctly in ios

### ISSUE THREAD

**Issue by Narendrastellent** [youtube/youtube-ios-player-helper#185]
1.Seek bar is not sliding smoothly. Can you please help me to solve this issue.
![simulator screen shot mar 15 2016 7 14 22 pm](https://cloud.githubusercontent.com/assets/13792735/13779741/8dd6114e-eae2-11e5-80f7-e63b3cbe2c5c.png)
2.I want to show the video played time and elapsed time same like default controller. Please suggest how to do with the custom seek bar if i hide the default controller.

**Comment by angelmic** [comment]
+1 I have same problem ~ help

**Comment by 180** [comment]
got the same problem

it works on iOS 8.4 and 9.3 but not 9.0, and 9.1

**Comment by Narendrastellent** [comment]
Yeah I am also facing same problem. so i can used for custom seekbar like
this.

Thanks & Regards.
   Narendra.

**Comment by designerdevil** [comment]
Any luck for the ios 9.0 Default seek bar fix?

**Comment by niaj-twinbit** [comment]
I have same problem ~ help
